config/.env.dev.txt

```
@@ -39,3 +39,4 @@ USE_REPLICA_DB=true
 GLIFIC_BASE_DOMAIN=glific.com
 # GLIFIC_API_HOST_OVERRIDE=https://your-ngrok-url.ngrok.io
 DIFY_API_KEY=app-this_is_not_the_key
+GUPSHUP_PARTNER_CLIENT_SECRET=this_is_not_secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GUPSHUP_PARTNER_CLIENT_SECRET key should go before the KAAPI_ENDPOINT key

(UnorderedKey)